### DISCUSSION TITLE
Browse Archives (tar, zip, etc)

### DISCUSSION THREAD

**Discussion by theduke** [spacedriveapp/spacedrive#2153] [Feature Request]
An awesome feature in a file explorer is the ability to browse the contents of archives directly in the explorer.

This is possible for all archives that support  browsing without actually extracting, like `.zip`, `.tar`, etc.

**Comment by utkubakir** [comment]
This is planned. We plan to support zipping/unzipping/browsing and all that jazz :)

**Comment by ketonik** [comment]
A more awesome feature is just to double click the damn thing and auto extract in place. ICG2fuks about seeing what's inside, I have to extract it anyway.
